endpoints.js

```diff
@@ -0,0 +1,17 @@
+/** "GET /api/articles": {
```

Comment:
is there a reason this file is commented out? I'd delete it if you're not using it